recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.0" %}
+{% set sha256 = "06ec0990d362963fc5e02ad64bc90d99ad0d226014354de4566af267d33443b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
+    - make
+    - go
+    - maven
+  host:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*
+    - llvmdev
+    - clangdev
+    - double-conversion
+    - snappy
+    - gflags
+    - glog
+    - libarchive
+    - libkml
+    - libpng
+    - blosc
+    - libgdal >=2.3
+    - arrow-cpp >=0.12.1
+    - thrift-cpp >=0.11.0
+    - ncurses
+    - flex
+    - bisonpp
+    - openssl
+    - librdkafka
+    - cudatoolkit-dev
+    - openldap
+    # Required by cudatoolkit-dev:
+    - wget
+  run:
+    - boost-cpp 1.68.*
```

Comment:
Why do you need this boost version? Doesn't it work with 1.70? 

As all other packages on conda-forge were migrated to 1.70, this will cause a lot of headaches.

Reply:
omniscidb 4.8 is not boost > 1.68 ready.